packages/block-editor/src/components/simulate-media-query/index.js

```diff
@@ -0,0 +1,128 @@
+/**
```

Comment:
I'd expect if the default export of this file is `useSimulatedMediaQuery`, the folder ought to be named accordingly, i.e. `use-simulated-media-query`.

Prior art:

- https://github.com/WordPress/gutenberg/tree/master/packages/compose/src/hooks
- https://github.com/WordPress/gutenberg/tree/master/packages/data/src/components